src/common.rs

```diff
@@ -16,16 +17,40 @@ impl Warning {
             line,
             check_name,
             message,
+            is_fixed: None,
         }
     }
+
+    pub fn line_number(&self) -> usize {
+        self.line.number
+    }
+
+    pub fn set_fixed(&mut self, val: bool) {
```

Comment:
```suggestion
    pub fn mark_as_fixed(&mut self, val: bool) {
```
I like more this name 🙂

Reply:
There are pretty many `set_` methods in Rust std library )
https://doc.rust-lang.org/std/net/enum.SocketAddr.html#method.set_ip
https://doc.rust-lang.org/std/env/fn.set_var.html
https://doc.rust-lang.org/std/fs/struct.File.html#method.set_len

But `mark_as_fixed` is good one, too.

Reply:
> There are pretty many `set_` methods in Rust std library )
> https://doc.rust-lang.org/std/net/enum.SocketAddr.html#method.set_ip
> https://doc.rust-lang.org/std/env/fn.set_var.html
> https://doc.rust-lang.org/std/fs/struct.File.html#method.set_len
> 
> But `mark_as_fixed` is good one, too.

I also think the `mark_as_fixed` is more clear, because it's a setter for a boolean variable :)

Reply:
I have renamed this method.